packages/@expo/config-plugins/src/android/BuildProperties.ts

```diff
@@ -72,6 +72,19 @@ export const withNewArchEnabledGradleProps = createBuildGradlePropsConfigPlugin<
   'withNewArchEnabledGradleProps'
 );
 
+/**
+ * A config-plugin to update `android/gradle.properties` from the `updates.useNativeDebug` in expo config
+ */
+export const withUpdatesNativeDebugGradleProps = createBuildGradlePropsConfigPlugin<ExpoConfig>(
+  [
+    {
+      propName: 'EX_UPDATES_NATIVE_DEBUG',
+      propValueGetter: (config) => (config?.updates?.useNativeDebug ?? false).toString(),
```

Comment:
```suggestion
      propValueGetter: (config) => config?.updates?.useNativeDebug?.toString(),
```

maybe try this so that even when the `updates.useNativeDebug` is not configured, we don't add a `EX_UPDATES_NATIVE_DEBUG=false` property.
unlike jsEngine and newArchEnabled, the `EX_UPDATES_NATIVE_DEBUG` feels like an advanced property and don't necessarily pull into gradle.properties/Podfile.properties.json

Reply:
I went with

```js
propValueGetter: (config) => (config?.updates?.useNativeDebug === true ? 'true' : undefined)
```